packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -406,6 +406,26 @@ export interface IContextualMenuItem {
    */
   role?: string;
 
+  /**
+   * Optional accessibility a list item's position in the list (aria-posinset) attribute that will be stamped on to the element.
+   */
+  positionInSet?: number;
```

Comment:
Once you add these in here, it's possible for the user to override them. Should they be allowed to override them? I feel like ContextualMenuItem has gotten too large and should be split. ContextualMenuItem is the information that gets passed into the contextmenu, then something like ContextualMenuItemProps are the props that will get passed to the component that renders the item.

Reply:
@dzearing any ideas?

Reply:
@joschect From my understanding, even if I didn't add those values in, wouldn't we still have the same problem, because of the any property field that already exists?

[propertyName: string]: any;

If that's the case, I think the only difference is whether or not we explicitly or implicitly inform the user that we have this information, or at least in this specific instance.

Reply:
I agree that the ContextualMenuItem is getting rather large, but I also think we can leverage the `[propertyName: string]: any` prop on IContextualMenuItem. The user could pass `'aria-posinset': <#> and 'aria-setsize': <#>` to menu items already and we should piggy back off of that when passing to onRender. This would allow to do everything we need without adding any new props

Reply:
If we went this route, the only thing that would need to be thought is if we want to allow creators to be able to override these values for non custom renders... Currently if a user supplies those props they will be overridden by the base render functions. If we want to allow this value to be overridden, we could do the spread (talked about in other comments) and pass the updated item to all render fuctions